addons/account/migrations/11.0.1.1/noupdate_changes.xml

```diff
@@ -0,0 +1,93 @@
+                        ${company.name}<br/>
+                        ${company.phone or ''}
+                    </td><td valign="middle" align="right" style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;">
+                        % if company.email:
+                        <a href="mailto:${company.email}" style="text-decoration:none; color: white;">${company.email}</a><br/>
+                        % endif
+                        % if company.website:
+                        <a href="${company.website}" style="text-decoration:none; color: white;">
+                        ${company.website}
+                        </a>
+                        % endif
+                    </td></tr></table></td></tr><tr><td align="center">
+            Powered by <a target="_blank" href="https://www.odoo.com">Odoo</a>.
+        </td></tr></tbody></table>
+</div>
+    </field>
+    <field name="name">Account: Invoice notification header</field>
+  </record>
+  <record model="mail.template" id="email_template_edi_invoice">
+    <field name="body_html" type="html">
+      <div><p>Dear ${object.partner_id.name}
+% set access_action = object.with_context(force_website=True).get_access_action()
+% set is_online = access_action and access_action['type'] == 'ir.actions.act_url'
+% set access_url = object.get_mail_url()
+
+% if object.partner_id.parent_id:
+    (<i>${object.partner_id.parent_id.name}</i>)
+% endif
+,</p><p>Here is, in attachment, your 
+% if object.number:
+invoice <strong>${object.number}</strong>
+% else:
+invoice
+% endif
+% if object.origin:
+(with reference: ${object.origin})
+% endif
+amounting in <strong>${object.amount_total} ${object.currency_id.name}</strong>
+from ${object.company_id.name}.
+</p>
+
+% if is_online:
+    <br/><br/>
+    <center><a href="${access_url}" style="background-color: #1abc9c; padding: 20px; text-decoration: none; color: #fff; border-radius: 5px; font-size: 16px;" class="o_default_snippet_text">View Invoice</a></center>
+% endif
+    <br/><br/>
+
+% if object.state=='paid':
+    <p>This invoice is already paid.</p>
+% else:
+    <p>Please remit payment at your earliest convenience.</p>
+% endif
+
+<p>Thank you,</p>
+<p style="color:#888888">
+% if object.user_id and object.user_id.signature:
+    ${object.user_id.signature | safe}
+% endif
+</p>
+</div>
+    </field>
+    <field name="name">Invoicing: Invoice email</field>
+  </record>
+  <record model="account.payment.term" id="account_payment_term_immediate">
+    <field name="note">Payment terms: Immediate Payment</field>
+  </record>
+  <record model="ir.sequence" id="sequence_payment_supplier_refund">
+    <field name="name">Payments supplier credit notes sequence</field>
+  </record>
+  <record model="account.payment.term" id="account_payment_term_15days">
+    <field name="note">Payment terms: 15 Days</field>
```

Comment:
Prefer not to reset payment terms, which may be modified. And nothing substantially was changed anyway compared to https://github.com/odoo/odoo/blob/10.0/addons/account/data/account_data.xml#L28